pkg/vintagewc/cluster.go

```diff
@@ -46,5 +50,5 @@ func (o Object) GetObject() client.Object {
 
 // on vintage WC, use extraconfig
 func (o Object) GetObservabilityBundleConfigMap() string {
-	return "observability-bundle-logging-values"
+	return "observability-bundle-logging-extraconfig"
```

Comment:
Is there a reason to remame it?

Reply:
No particular reason, just gives directly the information about the nature of the cm

Reply:
But in that case that brings information about how it's mounted instead of what it is about :)

Reply:
True but I guess it's less confusing for people to have really distinct name for those cms

Reply:
Yep, but maybe we should name it with something related to promtail-toggle then but maybe we can discuss that later :)